### ISSUE TITLE
Debugger ignores breakpoints for mjs modules

### ISSUE THREAD

**Issue by zaverden** [Microsoft/vscode-node-debug2#192]
Hello,

I have the same issue as  #158 (despite of it is marked as solved)

Here is my `index.mjs` file:
```js
const i = 5;
console.log("i: ", i);
```
I also have this script in my `package.json`:
```json
  "scripts": {
    "debug": "node --experimental-modules --nolazy --inspect-brk=29229 index.mjs"
  }
```
And here is my `launch.json` configurations:
```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "Launch node",
      "program": "${workspaceFolder}/index.mjs",
      "runtimeArgs": [
        "--experimental-modules"
      ]
    }, {
      "type": "node",
      "request": "launch",
      "name": "Launch via NPM",
      "runtimeExecutable": "npm",
      "runtimeArgs": [
        "run-script",
        "debug"
      ],
      "port": 29229
    }
  ]
}
```
So, I have tried these 2 configurations, both run, show me `i: 5` in console, but don't stop at breakpoints in the `index.mjs`. Am I missing some configuration attributes?

I'm using Windows 10 Pro
VS Code: v1.23.1
Node Debug: v1.23.5
Node JS: v9.11.1 and v10.1.0 (I have tried both)

**Comment by zaverden** [comment]
I'm able to debug mjs in google chrome using NiM extension. 

And I have found a way to do some debugging:
I have added new launch configuration
```json
    {
      "type": "node",
      "request": "attach",
      "name": "Attach node",
      "port": 29229
    }
```
then I've started an app by `npm debug` (see debug script above), then hit F5 - and now I can go through my code step by step. 
It works, but I can't say I can use it:
- it is 2 separate steps  to launch
- it opens my files in separate tab as "_read-only core module_", and it does not saves break points

So, right now NiM is the best debugging solution. But I really like vscode and I'd like to debug my code without switching a context to browser.

P.S.
I have also tried to turn on "Auto Attach" - it ignores breakpoints too
